bot/data/text/en.json

```diff
@@ -121,14 +121,13 @@
         "youtube": "`{0}youtube <search>` *searches on youtube for your query*",
         "image": "`{0}image <search>` *searches google images for your query*",
         "math": "`{0}math <problem>` *solves a math problem",
-        "remindme": "`{0}remindme <time> <reminder>` *reminds you in a specified time*",
         "credits": "`{0}credits` *shows who helped make Villager Bot*",
         "snipe": "`{0}snipe` *shows the latest deleted message in the current channel*",
         "redditdl": "`{0}redditdl <reddit post url>` *downloads the specified media from Reddit*",
         "exif": "`{0}exif` *extracts EXIF data from the specified image*",
         "imagetopng": "`{0}imagetopng *converts the uploaded/specified image to a png*",
         "translate": "`{0}translate <target> [text]` *translates the given text or message to the specified target language*",
-        "reminders": "`{0}reminders` *shows you all your current reminders*"
+        "reminders": "`{0}reminders` `add` *or* `remove` *to add or remove a reminder respectively*"
```

Comment:
You didn't make this change for the rest of the languages or am I tripping?

Reply:
Oh yeah you right

Reply:
https://github.com/Iapetus-11/Villager-Bot/pull/265/commits/a13517ae5b2ea500e10e2b6d8bf2e8f318a2bdba